src/doc/gui_manual/WorkingWithFiles/Read_Write_Options.rst

```diff
@@ -0,0 +1,184 @@
+.. _Read_Write_Options:
+
+Database Read and Write Options
+-------------------------------
+Several database plugins have options that effect reading and/or writing with that
+format. These are described in this section. Alternatively, in some cases, the
+behavior of a database plugin may be effected by enviornment variables.
+
+Chombo
+~~~~~~
+
+Exodus
+~~~~~~
+   
+
+Detect Compound Variables
+"""""""""""""""""""""""""
+
+Checking this option will cause the plugin to try to guess that similarly named
+variables are the scalar components of an aggregate type such as a vector,
+tensor or array variable. The plugin will then automatically define expressions
+for these aggregate typed variables. For example, it will cause the plugin to
+combine three scalar variables with names such as ``velx``, ``vely`` and ``velz``
+into a *vector* expression ``vel`` defined as ``{velx, vely, velz}``. Note that
+this is just a convenience to free users from having to define expressions
+manally within their VisIt_ session.
+
+
+Use Material Convention
+"""""""""""""""""""""""
+
+With this option, the user can cause the plugin to recognize standard or custom
+*material conventions*. The Exodus file format does not define any specific
+standards for handling advecting and *mixing* materials. Different data
+producers have defined different conventions. A few pre-defined conventions for
+handling mixed materials from Exodus files are supported. In addition, users can
+define their own custom conventions as well. For a custom convention, the user
+must define the *namescheme* that will produce the names of the scalar variables
+holding material volume fractions. Optionally, users can specify a namescheme to
+produce the names of the scalar variables holding material-specific values for an
+associated non-material-specific variable.
+
+The
+`nameschemes <https://wci.llnl.gov/content/assets/docs/simulation/computer-codes/silo/LLNL-SM-654357.pdf?#page=226>`_
+used here are identical to those described in the
+`Silo user's manual <https://wci.llnl.gov/content/assets/docs/simulation/computer-codes/silo/LLNL-SM-654357.pdf>`_
+with one extension. The conversion specifier ``%V`` is used to denote the basename
+(non-material-specific) name of a set of scalar variables holding material
+specific values.
+
+The ALEGRA nameschemes for volume fraction and material specific variables are
+``"@%s@n?'&VOLFRC_%d&n&':'VOID_FRC':@"`` and ``"@%V_%d@n"``.
```

Comment:
Not sure if this is intended to be formatted by sphinx in some way, but it pretty much turned out the way it is written here. Just wanted to confirm that was intended

Reply:
Yes, those are *literal* namescheme strings.